packages/storybook/src/Link/Link.stories.tsx

```diff
@@ -0,0 +1,26 @@
+import * as React from 'react'
+import { Link } from '@kodiak/primitives'
+
+export default { title: 'Link' }
+
+export function initial() {
+  return (
+    <Link href="https://jilt.com" target="_blank" rel="noreferrer">
+      jilt.com
+    </Link>
+  )
+}
+
+export function withHover() {
+  return (
+    <Link
+      href="https://jilt.com"
+      color="red"
+      sx={{ '&:hover': { color: 'blue' } }}
```

Comment:
@alexanderchan this can be added to the default styling so that the `sx` prop doesn't have to be used only for hover.

Reply:
That being said, it's fine to leave this here, but will prefer to use variants for this.

Reply:
Makes sense, I can try to port over some of the styling